From 170281f8d756f561b2ffe612e0076b6bfff5f482 Mon Sep 17 00:00:00 2001
From: Daniel Burgener <Daniel.Burgener@microsoft.com>
Date: Fri, 28 Aug 2020 13:32:31 +0000
Subject: [PATCH] Allow setting arguments to xmlto via environmental variable.

This will allow us to control xmlto behavior from the spec file
---
 Makefile | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/Makefile.original b/Makefile
index d4a1c35..c4fd438 100644
--- a/Makefile.original
+++ b/Makefile
@@ -40,10 +40,10 @@ $(SECIL2CONF): $(SECIL2CONF_OBJS)
 man: $(SECILC_MANPAGE) $(SECIL2CONF_MANPAGE)
 
 $(SECILC_MANPAGE): $(SECILC_MANPAGE).xml
-	$(XMLTO) man $(SECILC_MANPAGE).xml
+	$(XMLTO) $(XMLARGS) man $(SECILC_MANPAGE).xml
 
 $(SECIL2CONF_MANPAGE): $(SECIL2CONF_MANPAGE).xml
-	$(XMLTO) man $(SECIL2CONF_MANPAGE).xml
+	$(XMLTO) $(XMLARGS) man $(SECIL2CONF_MANPAGE).xml
 
 install: all man
 	-mkdir -p $(DESTDIR)$(BINDIR)
-- 
2.17.1

